moninag/static/src/ts/app/app-routing.module.ts

```diff
@@ -1,6 +1,7 @@
 import { HttpModule, JsonpModule } from '@angular/http';
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { FlexLayoutModule } from "@angular/flex-layout";
```

Comment:
Delete this import, `FlexLayoutModule` is already imported in `app.module.ts` :recycle: